src/components/Home.js

```diff
@@ -7,30 +7,48 @@ import MapViewContainer from "../containers/MapViewContainer";
 import store from "../store";
 import * as raceGroupActions from "../actions/raceGroupsActions";
 import {geolocated} from "react-geolocated";
+import * as riderStageConnectionsActions from '../actions/riderStageConnectionsActions'
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import of 'riderStageConnectionsActions'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1128)